main.js

```diff
@@ -1,17 +1,30 @@
-var $currentTime = document.getElementById('time')
-var $startbutton = document.getElementById('startbutton')
-var currentTime = parseInt($currentTime.textContent, 10)
+var $time = document.getElementById('time') // the variable that is controlling the time
+var $startPause = document.getElementById('startPause')
+var currentTime = parseInt($time.textContent, 10) // obtaining the content of time and making it a number
+var intervalId;
 
-function timeStart() {
+$startPause.addEventListener('click', function () {
 
-  var timeIncrease = setInterval(function () {
-    $currentTime.textContent = currentTime += 1
-  }, 1000)
-}
+  var runState = $startPause.textContent
 
-$startbutton.addEventListener('click', timeStart, false)
+  function increaseTime() {
+    currentTime += 1
+    $time.textContent = currentTime
+  }
+  // if Timer is paused: clear interval, start interval, chenge the button to "pause" & red
```

Comment:
You can omit journalling comments like this. Assume your reader knows JavaScript.

Reply:
These comments are great while your coding, but you should delete them before opening a PR.